Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -413,7 +413,7 @@
                                 </controls:LayoutTransformControl.Transform>
 
                                 <ToggleButton x:Name="PART_ExpanderToggleButton" 
-                                              Height="40"
+                                              MinHeight="40"
```

Comment:
This change looks unrelated

Reply:
This change is necessary if the Content is higher than 40 then the header should be higher.